microceph/api/cluster.go

```diff
@@ -0,0 +1,73 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"regexp"
+
+	"github.com/canonical/lxd/lxd/response"
+	"github.com/canonical/lxd/shared/logger"
+	"github.com/canonical/microceph/microceph/api/types"
+	"github.com/canonical/microceph/microceph/ceph"
+	"github.com/canonical/microceph/microceph/constants"
+	"github.com/canonical/microceph/microceph/interfaces"
+	"github.com/canonical/microcluster/rest"
+	"github.com/canonical/microcluster/state"
+)
+
+var clusterCmd = rest.Endpoint{
+	Path: "cluster",
+	Get:  rest.EndpointAction{Handler: cmdClusterGet, ProxyTarget: false},
+}
+
+// cmdClusterGet returns a json dump of the cluster config db.
```

Comment:
Nit: clarify doc string -- as stated it sounds a bit like this would return a dump of the complete DB

Reply:
yes, it did that but it got complicated when we started issuing each remote their own keys.
I will update the docstring to fix this.